src/strategies/naga/index.ts

```diff
@@ -0,0 +1,107 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+import { strategy as erc20BalanceOfStrategy } from '../erc20-balance-of';
+
+export const author = 'naga-finance';
+export const version = '0.0.1';
+
+const masterChefAbi = [
+  {
+    inputs: [
+      {
+        internalType: 'uint256',
+        name: '',
+        type: 'uint256'
+      },
+      {
+        internalType: 'address',
+        name: '',
+        type: 'address'
+      }
+    ],
+    name: 'userInfo',
+    outputs: [
+      {
+        internalType: 'uint256',
+        name: 'amount',
+        type: 'uint256'
+      },
+      {
+        internalType: 'uint256',
+        name: 'rewardDebt',
+        type: 'uint256'
+      }
+    ],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
+
+const masterChefContractAddress = '0x428C736EA96f71D5770C22f36De5c806D35c2F26'; //Masterchef
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+  const score = await erc20BalanceOfStrategy(
+    space,
+    network,
+    provider,
+    addresses,
+    options,
+    snapshot
+  );
+
+  const masterBalances = await multicall(
+    network,
+    provider,
+    masterChefAbi,
+    addresses.map((address: any) => [
+      masterChefContractAddress,
+      'userInfo',
+      ['0', address]
+    ]),
+    { blockTag }
+  );
+
+  const sousBalances = await Promise.all(
+    options.chefAddresses.map((item) =>
+      multicall(
+        network,
+        provider,
+        sousChefabi,
```

Comment:
This is undefined.. please try running this strategy locally.. use command:

`npm run test --strategy=naga`

Reply:
Okie